src/ProfileRight.php

```diff
@@ -45,6 +45,34 @@ class ProfileRight extends CommonDBChild
     public static $items_id = 'profiles_id'; // Field name
     public $dohistory       = true;
 
+    public function clone(array $override_input = [], bool $history = true)
```

Comment:
Could you explain why you overrided the `clone` mehod ?

Reply:
The default implementation uses an `add` call to create the cloned relations. In the case of profiles, the the rights are added when the Profile is created with values of 0 so we have to do an update or insert to properly "clone" the rights from the original Profile.

Reply:
Could you add a comment in code ?